functional_tests/internal/common.go

```diff
@@ -195,3 +201,12 @@ func WaitForTerminatingPods(t *testing.T, clientset *kubernetes.Clientset, names
 		return terminatingPods == 0
 	}, 2*time.Minute, 5*time.Second, "there are still terminating pods after 2 minutes")
 }
+
+func DeleteObject(t *testing.T, k8sClient *k8stest.K8sClient, objYAML string) {
```

Comment:
There were two identical `deleteObject` methods in `istio_test.go` and `k8sevent_test.go`, so I've consolidated them here.